tests/periph_i2c/tests/01__periph_i2c_base.robot

```diff
@@ -34,3 +34,9 @@ Read Register After NACK Should Succeed
     API Call Should Error       I2C Read Reg  addr=42
     API Call Should Succeed     I2C Read Reg
     API Call Should Succeed     I2C Release
+
+    Multiple Read Byte NACKs Should Recover
```

Comment:
naming is off compared to previous test, maybe `Read Byte After multiple NACKs should succeed/recover`